docs/04-resources/06-guides/powpeg/advanced-operations/index.md

```diff
@@ -2,38 +2,38 @@
+description: "See how to perform advanced operations on the PowPeg App"
 tags: [2 way peg app, powpeg, peg-in, peg-out, bridge, rsk, rootstock]
 ---
 
-This section contains detailed instructions on how to perform advanced operations on the 2 way peg app. 
+This section contains detailed instructions on how to perform advanced operations on the PowPeg app. 
 
 These operations include;
 
-* How to review funds in Bitcoin after a pegout by [viewing a derived address](/resources/guides/two-way-peg-app/pegout/deriving-electrum), 
-* Convert RBTC - BTC and import a [key in Electrum](/resources/guides/two-way-peg-app/pegout/deriving-electrum#import-key-in-electrum), import in Electrum if you are using [hardware wallets](/resources/guides/two-way-peg-app/pegout/deriving-electrum#import-key-in-electrum-using-hardware-wallets)
+* How to review funds in Bitcoin after a pegout by [viewing a derived address](/resources/guides/powpeg/pegout/deriving-electrum), 
+* Convert RBTC - BTC and import a [key in Electrum](/resources/guides/powpeg/pegout/deriving-electrum#import-key-in-electrum), import in Electrum if you are using [hardware wallets](/resources/guides/powpeg/pegout/deriving-electrum#import-key-in-electrum-using-hardware-wallets)
 * Selecting different accounts
 * Viewing advanced details
 * Adjusting network fees
 * Viewing Advanced data
 
-> For  more information, see [design and architecure](/resources/guides/two-way-peg-app/advanced-operations/design-architecture/), [supported addresses](/resources/guides/two-way-peg-app/advanced-operations/supported-addresses/), [supported wallets](/resources/guides/two-way-peg-app/advanced-operations/supported-wallets/), [supported browsers](/resources/guides/two-way-peg-app/advanced-operations/supported-browsers/)
+> For  more information, see [design and architecure](/resources/guides/powpeg/advanced-operations/design-architecture/), [supported addresses](/resources/guides/powpeg/advanced-operations/supported-addresses/), [supported wallets](/resources/guides/powpeg/advanced-operations/supported-wallets/), [supported browsers](/resources/guides/powpeg/advanced-operations/supported-browsers/)
 
 --- 
 
 ## Account selection
 
 ### Pegin:
 
-There are three types of accounts on the 2 way peg app. See [supported addresses](/resources/guides/two-way-peg-app/advanced-operations/supported-addresses/) section for examples of these types of addresses.
+There are three types of accounts on the PowPeg app. See [supported addresses](/resources/guides/powpeg/advanced-operations/supported-addresses/) section for examples of these types of addresses.
 
 To select an account to send BTC from, click on **Select the account** as shown in the image below. This loads the balance for all the addresses in your hardware wallet.
 
-> Note: Your hardware wallet must be connected to view this section of the 2 way peg app.
+> Note: Your hardware wallet must be connected to view this section of the PowPeg.
```

Comment:
application name should be: "PowPeg App"

Reply:
Updated too